### ISSUE TITLE
[Documentation] Mention the difference between commands and services

### ISSUE THREAD

**Issue by ShalokShalom** [habitat-sh/habitat#2914]
Currently suggests the documentation that commands like pkg-config get launched inside the studio with `hab sup start ..` since their installation process is the same like for services, which suggests, that they get launched as such ones, while they get launched via `hab pkg exec core/pkg-config pkg-config ..`

Is it valid to mention that here? https://www.habitat.sh/docs/run-packages-overview/

**Comment by eeyun** [comment]
Yep, I think this touches on a need for some more clear documentation around how to execute binaries out of habitat packages that are binary only as opposed to something that is a service! Good call!

**Comment by ShalokShalom** [comment]
What do you think? Does it make sense to set services like pkg-config by default to binlink?
I create a new issue for it if you agree. I think most people use pkg-config and a bunch of other commands and they are used to use them by simply typing their name. 

According to Habitats goal "Provide a low barrier to entry" can this help to feel quickly like in a common environment. 

While I guess that this decision might make this quite unrealistic: 
https://github.com/habitat-sh/habitat/pull/2915#pullrequestreview-56243317

**Comment by kagarmoe** [comment]
Agree. We don't explain how to execute binaries out of habitat packages.

**Comment by ShalokShalom** [comment]
Is this still an issue?
